src/linear_algebra/finrank.lean

```diff
@@ -117,14 +108,45 @@ begin
   exact hxy (subsingleton.elim _ _)
 end
 
+end
+
+
+section
+variables [strong_rank_condition K]
+
+/-- If a vector space has a finite basis, then its dimension is equal to the cardinality of the
```

Comment:
Maybe this should say module rather than vector space now (or both)

Reply:
Done